pkg/operator/api/userconfig/apis.go

```diff
@@ -80,12 +100,23 @@ func (apis APIs) Validate() error {
 }
 
 func (api *API) Validate() error {
-	if api.ModelPath == nil && api.Model == nil {
-		return errors.Wrap(ErrorSpecifyOnlyOneMissing("model_name", "model_path"), Identify(api))
+	if api.ExternalModel == nil && api.Model == nil {
+		return errors.Wrap(ErrorSpecifyOnlyOneMissing("model", "external_model"), Identify(api))
 	}
 
-	if api.ModelPath != nil && api.Model != nil {
-		return errors.Wrap(ErrorSpecifyOnlyOne("model_name", "model_path"), Identify(api))
+	if api.ExternalModel != nil && api.Model != nil {
+		return errors.Wrap(ErrorSpecifyOnlyOne("model", "external_model"), Identify(api))
+	}
+
+	if api.ExternalModel != nil {
+		bucket, key, err := aws.SplitS3Path(api.ExternalModel.Path)
+		if err != nil {
+			return err
```

Comment:
I think we should wrap err with `Identify(api)` and the name of the field, so `"external_model", "path"`